src/pages/nft/class/_classId/index.vue

```diff
@@ -307,10 +293,10 @@ export default {
       this.uiSetTxStatus('');
       this.fetchUserCollectedCount();
     },
-    async onTransfer() {
-      logTrackerEvent(this, 'NFT', 'NFTTransfer(DetailsPage)', this.classId, 1);
+    async onTransfer(nftId) {
```

Comment:
It will be picked by user in Transfer dialog
https://github.com/likecoin/liker-land/blob/793d1ff55281d3ba07b352757754df5a31f92cc2/src/components/EventModal/Transfer.vue#L160

Reply:
I see, and how about here?
https://github.com/likecoin/liker-land/blob/9e29aafbef33c8a706b3ca47424e33adb0597eb5/src/pages/nft/class/_classId/index.vue#L109